frontend/packages/ux-editor-v3/src/testing/schemas/json/component/Accordion.schema.v1.json

```diff
@@ -1,19 +1,53 @@
 {
```

Comment:
From my point of view, there are too many changes to the V3 app schemas. I believe that Apps has not made that many changes on v3 after the v4 release. 🤔

Reply:
Just looked into this 😊 

1. Some properties have switched places. For example, `headingLevel` was previously located above `grid`. It is now the opposite.
2. The script has since 16.02.2024 auto-expanded the `$ref` pointers within the JSON schema (see below).

___
### **v3 Schemas as of 01.02.2024**

The v3 component schemas were last updated on **February 1, 2024**, as seen in this [commit](https://github.com/Altinn/altinn-studio/commit/e13cf0a3a9eb418ddd71504f94b705995a05bb53). In this update, the schemas used `$ref` pointers to reference definitions elsewhere in the schema. For example:

```json
"grid": {
  "$ref": "#/definitions/IGrid"
},
```

### **Updates on 16.02.2024: Introduction of `expandRef` Function**

On **February 16, 2024**, a new function called `expandRef` was introduced. This function automatically expands all `$ref` pointers within the JSON schema, effectively inlining the referenced definitions.

**Before `expandRef` Function:**

```json
"grid": {
  "$ref": "#/definitions/IGrid"
},
```

**After `expandRef` Function:**

```json
"grid": {
  "properties": {
    "xs": { "$ref": "#/definitions/IGridSize" },
    "sm": { "$ref": "#/definitions/IGridSize" },
    "md": { "$ref": "#/definitions/IGridSize" },
    "lg": { "$ref": "#/definitions/IGridSize" },
    "xl": { "$ref": "#/definitions/IGridSize" },
    "labelGrid": { "$ref": "#/definitions/IGridStyling" },
    "innerGrid": { "$ref": "#/definitions/IGridStyling" }
  }
},
```

___
Will this affect how Studio works? I'll do some manual testing and find out 😊

Reply:
We should test with a v3 app. :)

Reply:
Tested with a local v3 app, and it appears to work correctly.
But we are excluding the v3 schemas from this PR in order to minimize the changes done in this iteration. 😊